apps/configurator/lib/configurator_web/controllers/character_controller.ex

```diff
@@ -70,4 +68,9 @@ defmodule ConfiguratorWeb.CharacterController do
     |> put_flash(:success, "Character deleted successfully.")
     |> redirect(to: ~p"/characters")
   end
+
+  defp get_curse_skills_by_type() do
+    GameBackend.Units.Skills.list_curse_skills()
```

Comment:
If for some reason we move the GameBackend application and split it from the Configurator this would crash, right?

Reply:
Yes! We would have to do a Finch request instead! Like we do with Arena & Configurator.

Reply:
Thanks!